app/assets/stylesheets/application.css.scss

```diff
@@ -23,17 +23,35 @@ body {
 
 .application-body{
   background-image: url("http://i.imgur.com/9olXuLe.jpg");
-  opacity: 0.97;
 }
 
 .application-body-doom{
   background-image: url('doom.jpeg');
-  opacity: 0.97;
   background-size:     cover;
   background-repeat:   no-repeat;
   background-position: center center;
 }
 
+.application-body-shire{
+  background-image: url('shire.jpeg');
+  background-size:     cover;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size<br>Colon after property should be followed by one space